storage/storage.go

```diff
@@ -264,10 +266,7 @@ func (storage DBStorage) WriteReportForCluster(
 		query = `INSERT OR REPLACE INTO report(org_id, cluster, report, reported_at, last_checked_at) 
 		 VALUES ($1, $2, $3, $4, $5)`
 	case DBDriverPostgres:
-		query = `INSERT INTO report(org_id, cluster, report, reported_at, last_checked_at)
-		 VALUES ($1, $2, $3, $4, $5)
-		 ON CONFLICT (org_id, cluster) 
-		 DO UPDATE SET report = $3, reported_at = $4, last_checked_at = $5`
+		query = `INSERT INTO report(org_id, cluster, report, reported_at, last_checked_at) VALUES ($1, $2, $3, $4, $5) ON CONFLICT (org_id, cluster)  DO UPDATE SET report = $3, reported_at = $4, last_checked_at = $5`
```

Comment:
why did you rewrite it to one huge line?

Reply:
Accident, fixed.